src/main/java/org/spongepowered/api/plugin/Plugin.java

```diff
@@ -41,23 +41,17 @@
     /**
      * An ID for the plugin to be used internally. The ID should be unique as to
      * not conflict with other plugins.
-     *
-     * @return The plugin identifier
```

Comment:
If you remove this the JDK will start complaining, so it doesn't really fix this issue currently. That's a bug in checkstyle, the return statement should be there